buildchow/templates/product-template.yaml

```diff
@@ -117,39 +109,44 @@
             - maven.test.error.ignore=true
             - download.cache.directory={download-cache}
             - targetPlatformGroup=jbdevstudio
-            - jbosstools-site=http://download.jboss.org/jbosstools/updates/nightly/core/${{jbosstools_site_stream}}/
+            - jbosstools-site=http://download.jboss.org/jbosstools/mars/snapshots/builds/jbosstools-build-sites.aggregate.site_${{jbosstools_site_stream}}/latest/all/repo/
           pom: ${{WORKSPACE}}/sources/product/site/pom.xml          
       - maven-target:
-          goals: "org.apache.maven.plugins:maven-dependency-plugin:2.9:unpack"
-          maven-version: maven-3.x
+          goals: "-U org.apache.maven.plugins:maven-dependency-plugin:2.9:unpack"
```

Comment:
why the -U ? 

At what point is it necessary to check that 2.9 is actually still 2.9 ? Don't -U only make sense for builds dependent on snapshots ?